obs-studio-server/source/nodeobs_api.cpp

```diff
@@ -495,8 +495,15 @@ void OBS_API::OBS_API_initAPI(
 	obs_data_set_bool(private_settings, "BrowserHWAccel", true);
 	obs_apply_private_data(private_settings);
 	obs_data_release(private_settings);
+	
+	int videoError;
+	if (!openAllModules(videoError)) {
+		rval.push_back(ipc::value((uint64_t)ErrorCode::Error));
+		rval.push_back(ipc::value(videoError));
+		AUTO_DEBUG;
+		return;
```

Comment:
We should return something to the frontend here, either a basic false value or an error code to tell the frontend something bad happened and let it handle it. What the frontend should probably do : popup a message to the user tell them what failed and what should be done.

Reply:
Currently the client will receive the videoError (int) from the server . Since I'm returning the message as an error the 	ValidateResponse() method, on the client side, will throw the error to the JS, this throw can be handled by the frontend or I need to perform this using some callback?

Reply:
`OBS_service_resetVideoContext` is used by the frontend in other places so it's good that it returns an error code now. But how it's written in your PR, OBS_API is not returning anything to the frontend. Both of of the functions need to return here. You can export an enum error code and use it for both.